cmd/api/src/database/graphschema.go

```
@@ -386,39 +429,68 @@ func (s *BloodhoundDB) DeleteGraphSchemaProperty(ctx context.Context, propertyID
 
+// GetGraphSchemaEdgeKinds - returns all rows from the schema_edge_kinds table that matches the given model.Filters. It returns a slice of model.GraphSchemaEdgeKinds
+// populated with data, as well as an integer indicating the total number of rows returned by the query (excluding any given pagination).
 func (s *BloodhoundDB) GetGraphSchemaEdgeKinds(ctx context.Context, edgeKindFilters model.Filters, sort model.Sort, skip, limit int) (model.GraphSchemaEdgeKinds, int, error) {
 	var (
 		schemaEdgeKinds = model.GraphSchemaEdgeKinds{}
 		totalRowCount   int
 	)
 
+	for column := range edgeKindFilters {
+		if column == "name" {
+			column = fmt.Sprintf("%s.%s", kindTable, column)
```

golangci-lint (2.5.0)

[major] ineffectual assignment to column

(ineffassign)